openshift/opendistro/docker/Dockerfile-es

```diff
@@ -0,0 +1,4 @@
+FROM amazon/opendistro-for-elasticsearch:0.8.0
```

Comment:
Let's add a comment to outline why we are "overriding" the default elastic. I.E. We need to run as a specific user for the cluster in the classified environment, etc

Reply:
I have added comments to the docker files.  The other two are for completeness for I use them as is.

Would like to convert all bash shell scripts to use ansible.  The big reason is I can specify variables in one location and be done  for bash shell if you want to override you have to repeat for each shell script